test/README.md

```diff
@@ -412,6 +412,9 @@ The realpath of the 'tmp' directory.
 
 Name of the temp directory used by tests.
 
+### childShouldNotThrowAndAbort()
+ This test makes sure that when using `--abort-on-uncaught-exception` and when throwing an error from within a domain that has an error handler setup, the process _does not_ abort.
```

Comment:
I meant a blank line between `### childShouldNotThrowAndAbort()` and the following paragraph. Also, lines should be 80 characters long. I'm not sure what you mean by temp directory.